src/cljc/harja/domain/roolit.cljc

```diff
@@ -39,42 +39,14 @@
 (def ely-urakanvalvoja "ELY_Urakanvalvoja")
 (def ely-paakayttaja "ELY_Paakayttaja")
 
-
-(def toteumien-kirjaus
```

Comment:
Täällä oli vanhaa käyttämätöntä koodia, ajalta jolloin luultiin että Harjaan tehdään oma käyttäjien hallinta... poistelen näitä vihdoin, ei ollut usageja.